tools/dev/tests/test_poetrypkg.py

```diff
@@ -0,0 +1,61 @@
+import subprocess
+import sys
+
+import pytest
+
+from pathlib import Path
+
+from click.testing import CliRunner
+
+cli = Path(__file__).parent.parent / "poetrypkg.py"
+with open(cli, encoding='utf-8') as f:
+    exec(f.read())
```

Comment:
this was following the import pattern in test_changelog.py because the tools/dev directory isn't a module: https://github.com/cloud-custodian/cloud-custodian/blob/master/tools/dev/tests/test_changelog.py#L11-L13